starlite/exceptions.py

```diff
@@ -14,9 +14,9 @@
 
 
 class StarLiteException(Exception):
-    def __init__(self, detail: str = ""):
+    def __init__(self, *args: Any, detail: str = "", **kwargs: Dict[str, Any]):
         self.detail = detail
-        super().__init__()
+        super().__init__(*args, **kwargs)
 
     def __repr__(self) -> str:
```

Comment:
Can we get rid of custom repr as well? `Exception.__repr__` does `StarLiteException('Message')` which is more conventional + a valid python syntax.

Reply:
I followed here the StarletteException pattern - if we dont override it, we will be left with their override in place:


```python
class HTTPException(Exception):
    def __init__(
        self,
        status_code: int,
        detail: typing.Optional[str] = None,
        headers: typing.Optional[dict] = None,
    ) -> None:
        if detail is None:
            detail = http.HTTPStatus(status_code).phrase
        self.status_code = status_code
        self.detail = detail
        self.headers = headers

    def __repr__(self) -> str:
        class_name = self.__class__.__name__
        return f"{class_name}(status_code={self.status_code!r}, detail={self.detail!r})"
```

This is their code

Reply:
Waiting on your approval or further suggestions 😉

Reply:
Which is fine? It'll only be in StarliteHTTPException. Why do we have to override it?

Reply:
Because we inherit it

Reply:
Sorry, I'm not following. I've updated original question, hope it does reads clearer now

Reply:
Lemme suggest you submit a PR with some tests and if all is good, we merge it?

Reply:
Don't have capacity for that at the moment

Reply:
NP. So let's keep it as is until you do 😉

Reply:
It is up to you really. Though, it's still unclear why we have to override, if we inherit it? And if we do, what you expect me to do in PR?

Reply:
Remove repr, test it, and assert the result is satisfactory and useful